src/components/TextFieldMedBredde.tsx

```diff
@@ -0,0 +1,45 @@
+import styled, { CSSObject } from 'styled-components/macro';
+import { TextField } from '@navikt/ds-react';
+
+type bredde = 'fullbredde' | 'XXL' | 'XL' | 'L' | 'M' | 'S' | 'XS' | 'XXS';
+
+interface Props {
+  bredde?: bredde;
+}
+
+export const TextFieldMedBredde = styled(TextField)<Props>(
+  (props: Props): CSSObject => {
+    return props.bredde ? breddeTilStyle[props.bredde] : {};
+  }
+);
+
+const breddeTilStyle: Record<bredde, CSSObject> = {
```

Comment:
Siden teksten på de nye komponentene er større må nok disse størrelsene justere litt oppover. Ellers risikerer vi unaturlige text-breaks slik som på det første etter-bildet.

Reply:
Har justert breddene nå for å ta hensyn til litt større font. Har sett på samtlige inputs som endres, og alle ser greie ut.